app/src/main/java/chat/rocket/android/chatroom/presentation/ChatRoomPresenter.kt

```diff
@@ -472,7 +473,7 @@ class ChatRoomPresenter @Inject constructor(
                         view.showInvalidFileMessage()
                     } else {
                         val byteArray =
-                            bitmap.getByteArray(mimeType, 100, settings.uploadMaxFileSize())
+                            bitmap.getByteArray(mimeType, 60, settings.uploadMaxFileSize())
```

Comment:
@peter it looks to me like getByteArray only gets called when you take a picture with the camera, and not when uploading files from the device, even if the files are jpgs, for example. Is this your intention?